### PR TITLE
aireplay-ng: add German translation

### PR DESCRIPTION
**Pull request by Slotharmy**
Translated the current aireplay-ng.md into german.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/aireplay-ng.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# aireplay-ng
+
+> Packete in ein WLAN injizieren.
```
Comment (pixelcmtd) [inline pages.de/common/aireplay-ng.md:3]:
```suggestion
> Pakete in ein WLAN injizieren.
```

File: pages.de/common/aireplay-ng.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+> Teil von `aicraick-ng`.
```
Comment (pixelcmtd) [inline pages.de/common/aireplay-ng.md:4]:
```suggestion
> Teil von `aircrack-ng`.
```

**Review by pixelcmtd — CHANGES_REQUESTED** [review]

File: pages.de/common/aireplay-ng.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+> Weitere Informationen: <https://www.aircrack-ng.org/doku.php?id=aireplay-ng>.
+
+- Senden einer bestimmten Anzahl an deauthentifizierungs Packeten mit der MAC-Adresse des Access POints, der MAC-Adresse des Clients und seines kabellosen interfaces:
```
[the remark this answers is not in the document]
Comment (Slotharmy) [reply pages.de/common/aireplay-ng.md:7]: In my 10 years working in studying and working, i have never heard of a Dissoziationspaket. But very surely i heard about a Deauthentifizierungspaket. I aggree with u on the rest of the changes but noch on the Dissoziationspaket part.

[the remark this answers is not in the document]
Comment (pixelcmtd) [reply pages.de/common/aireplay-ng.md]: Is „Deauthentifizierungspaket“ the German term used for an 802.11 disassociate packet tho?

**Review by kbdharun — CHANGES_REQUESTED** [review]
Seems like you already added the page [here](https://github.com/tldr-pages/tldr/blob/main/pages.de/common/aircrack-ng.md) in #9938, feel free to merge the current PRs branch with main and update it with the suggestions here.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.de/common/aireplay-ng.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Slotharmy) [reply pages.de/common/aireplay-ng.md]: yes it is

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**